.github/workflows/on_propen_push_do_test.yml

```diff
@@ -0,0 +1,72 @@
+name: On PR Open or Push Do Build and Test
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - 15.0.x
```

Comment:
Maybe we should have a wildcard for the minor release here as it's probable that we will end up with multiple 15 minor branches.